README.md

```diff
@@ -555,26 +557,47 @@ These are the steps for manual set-up of certificates:
    $ KUBCONFIG="<<your cluster kubeconfig path>> PATH="$PATH:$PWD/_work/bin" ./test/setup-ca-kubernetes.sh
 ```
 
-- **Deploy the driver to Kubernetes using DeviceMode:LVM**
+- **Deploy the driver to Kubernetes**
 
-```sh
-    $ sed -e 's/PMEM_REGISTRY/<your registry>/' deploy/kubernetes-<kubernetes version>/pmem-csi-lvm.yaml | kubectl create -f -
-```
+The `deploy/kubernetes-<kubernetes version>` directory contains
+`pmem-csi*.yaml` files which can be used to deploy the driver on that
+Kubernetes version. The files in the directory with the highest
+Kubernetes version might also work for more recent Kubernetes
+releases.
+
+For each Kubernetes version, four different deployment variants are provided:
+
+-- `direct` or `lvm`: one uses DeviceMode:Direct, the other Device:LVM
```

Comment:
- Device:LVM should be DeviceMode:LVM
- production should be inside single quotes as well?
- double-dash as startmark does not work as intended, result is two lines do not appear on separate lines (spotted  in preview)

Reply:
>    Device:LVM should be DeviceMode:LVM

Fixed.

>    production should be inside single quotes as well?

No, it's not a literal part of the filenames. I wanted to highlight that there are again two variants, but that wasn't obvious. Now it just lists `testing` - is that clearer?

>   double-dash as startmark does not work as intended, result is two lines do not appear on separate lines (spotted in preview)

That was meant to be sub-items. The correct syntax is indented instead of double dashes,  but I am not sure whether that gets rendered as such now - probably not. I kept the indented dashes nevertheless because it reads better in the README.md source.